providers/plugin.rb

```diff
@@ -14,7 +14,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-use_inline_resources if defined?(:use_inline_resources)
+use_inline_resources if defined?(:use_inline_resources) # ~FC113
```

Comment:
We can just set use_inline_resources. Checking if it's defined is a Chef 11 thing